source/includes/_webhooks.md

```diff
@@ -452,12 +452,13 @@ This is fired whenever a wallet is created.
 
 ```json
 {
-  "walletId" : "string",
+  "id" : "3d9ca033-eb05-459f-9f70-1139d2e2b213",
```

Comment:
Are Nester ok with this change? At least do we need to prep them for it?

Reply:
I'll tell them it changed.
Webhooks aren't working atm (still in review).